README.md

```diff
@@ -0,0 +1,34 @@
+This code is used to build and publish a docker image that the Single Cell Portal team uses as a basis for Rails servers.
+
+# What is this this? Why does it exists? #
+
+This provides a base image with ruby on rails ready to go with phusion passenger. The single cell portal has a docker image that was formerly built on `phusion/passenger-docker`, but this ensures that it's built on a base image provided by Google, which is good because they keep those images up-to-date with security fixes and it apparently makes our auditors happy. So here, we rebuild a couple images from phusion with tweaks to make that happen.
+
+# using the image #
+
+This image is available as [`singlecellportal/rails-baseimage` on Dockerhub](https://hub.docker.com/r/singlecellportal/rails-baseimage), so specify that in your docker commands or Dockerfile.
+
+# To make changes and publish a new image... #
+
+## making changes ##
+
+`./Dockerfile` is the main docker file as you would expect, so that's the file to modify to change what's in the image.
+
+To update the versions of the underlying images, modify requirements.bash (and the Dockerfile). We are specifying a docker image version for the underlying base image, but the version numbers for the phusion images actually correspond to github release tags, since we are rebuilding them from (slightly altered) source.
```

Comment:
```suggestion
To update the versions of the underlying images, modify requirements.bash (and the Dockerfile). We are specifying a Docker image version for the underlying base image, but the version numbers for the Phusion images actually correspond to GitHub release tags, since we are rebuilding them from (slightly altered) source.
```